mmv1/third_party/terraform/tests/resource_compute_instance_group_manager_test.go.erb

```diff
@@ -1033,7 +1034,6 @@ resource "google_compute_instance_group_manager" "igm-rolling-update-policy" {
   target_size        = 3
   update_policy {
     type                  = "PROACTIVE"
-    minimal_action        = "REPLACE"
```

Comment:
Minimal action needs to be added back here

Reply:
ditto

Reply:
I reverted this change. Lets decouple this discussion from this PR.